.github/workflows/latex-linter.yaml

```diff
@@ -0,0 +1,13 @@
+name: run-linters
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+    paths: ["docs/**.tex"]
+jobs:
+  run-linter:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+      - run: cargo install tex-fmt
```

Comment:
kind of bad practice at some point I'll initialize this repo as a rust project with a dummy, lib.rs or main.rs, and make a lockfile to install the dependencies from, but I'm being a bit lazy rn

Reply:
also this stuff is cached by the rust toolchain action, so don't worry about reinstall every time